public/app/core/components/TimelineChart/utils.ts

```diff
@@ -283,7 +283,7 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<UPlotConfigOptions> = (
     let cursor: Partial<uPlot.Cursor> = {};
 
     cursor.sync = {
-      key: '__global_',
+      key: eventBus.id,
```

Comment:
I just tested setting this to uuidv4() (ie a new guid every time), and everything still works :) 

Btw you only changed this on the TimeLineChart, the one for TimeSeries is under TimeSeries/utils

Reply:
cc @leeoniya

Reply:
weird